rest/src/main/java/no/nav/vedtak/felles/prosesstask/rest/ProsessTaskRestTjeneste.java

```diff
@@ -79,7 +74,8 @@ public ProsessTaskRestTjeneste(ProsessTaskApplikasjonTjeneste prosessTaskApplika
     @BeskyttetRessurs(actionType = ActionType.CREATE, property = ABAC_DRIFT_ATTRIBUTT)
     public ProsessTaskDataDto createProsessTask(@Parameter(description = "Informasjon for restart en eksisterende prosesstask") @TilpassetAbacAttributt(supplierClass = AbacEmptySupplier.class) @Valid ProsessTaskOpprettInputDto inputDto) {
         // kjøres manuelt for å avhjelpe feilsituasjon, da er det veldig greit at det blir logget!
-        LOG.info("Oppretter prossess task av type {}", inputDto.getTaskType());
+        var sanitizedTaskType = inputDto.getTaskType().replace("\n", "").replace("\r", "").trim();
```

Comment:
Du kunne brukt LoggerUtils.toStringWithoutLineBreaks() fra felles-log for dette.